copc/lidar-las-to-copc.ipynb

```diff
@@ -0,0 +1,857 @@
+{
```

Comment:
<p>Link documentation to explore operations? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Thanks</p>